msal/tests/Test.MSAL.NET.Integration/UsernamePasswordIntegrationTests.cs

```diff
@@ -0,0 +1,139 @@
+//----------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+using Microsoft.Identity.Client;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Net;
+using System.Security;
+using System.Threading.Tasks;
+using Test.Microsoft.Identity.Core.UIAutomation.infrastructure;
+using Test.MSAL.NET.Integration.Infrastructure;
+
+namespace Test.MSAL.NET.Integration
+{
+    [TestClass]
+    public class UsernamePasswordIntegrationTests
+    {
+        public const string ClientId = "0615b6ca-88d4-4884-8729-b178178f7c27";
+        public const string Authority = "https://login.microsoftonline.com/organizations/";
+        public string[] Scopes = { "user.read" };
```

Comment:
>user.read [](start = 36, length = 9)

should this be User.Read or doesn't it matter?

Reply:
sure. done.

---
In reply to: [220424874](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1271#discussion_r220424874) [](ancestors = 220424874)